docs/docker.md

```diff
@@ -67,40 +71,80 @@ There are three kinds of guidelines included:
     injected as part of the Docker build. Dockerfiles **SHOULD NOT** copy
     default configuration directly from GitHub.
 
--   A GE's Docker **MUST** be published at least under the 'fiware' account on
-    DockerHub (if possible, please avoid the term 'fiware' in your repository
-    name, as the username already contains it).
-
-The GE owner will be responsible for publication and maintenance operations. The
-publication method will be through the 'Automatic Build Procedure' which allows
-to link a GitHub repository to a Dockerhub one (and keep them in sync). In order
-to get access to that account and proceed with the publication, please get in
-touch with the FIWARE team at
-[fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org).
-
--   Dockerfiles and Dockerhub repositories **MUST** be linked from the FIWARE
-    Catalogue according to the Guidelines defined by the
-    [FIWARE Catalogue](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Working_with_the_FIWARE_catalogue#Creating_instances).
-
 -   The GitHub repository `README.md` **MUST** have a Docker reference - this is
     a link on the mandatory Docker Pulls `README.md` badge
 
 -   The **ReadtheDocs** GE Installation Documentation **MUST** include
     references to the Docker Hub image, and how to configure it.
 
+## Publication Requirements
+
+-   The GE owner will be responsible for Docker publication and maintenance
+    operations. Docker builds **SHOULD** be automated. GE Owner's build should
+    be triggered by git push, FIWARE's one will be triggered by the release of
+    new FIWARE-tagged versions.
```

Comment:
> GE Owner's build  should be triggered by git push so `latest` will indicate the development tip. (**note** as explained below, this is not the same as the FIWARE equivalent which will only be triggered by the release of tagged versions.  This implies the `latest` release on the `/r/fiware` will always represent a stable build.)

I think we need to build all tagged releases, not only FIWARE tagged releases, if an interim release occurs we should mirror it on the `/r/fiware` account.

Reply:
mmm that sounds a bit contradictory to me ... Can you explain why do you think so?

Reply:
Users may want to test the use of bleeding edge development - this should not be a FIWARE responsibility.

FIWARE should only provide stable tested quality builds
Ideally we will start smoke testing CB compatibility builds before tagging on our account as well.

This approach will then match the approach taken in the GitHub catalogue on the `release` branch -  https://github.com/Fiware/catalogue/tree/releases - only release builds allowed.